_assets/js/main.js

```diff
@@ -857,18 +858,35 @@ jQuery(document).ready(function($) {
           .find('button.close')
           .off('click')
           .on('click', function() {
-            $panel.velocity('transition.slideDownOut', {
-              duration: 300,
-              complete: function() {
-                if (!$body.hasClass('using-mouse')) {
-                  $link.focus();
-                }
-              },
-            });
+            closePanel($link, $panel);
+          });
+        $(document)
+          .off('click keyup')
```

Comment:
This removes all document click/keyup handlers, which might be confusing if others are added. Maybe it makes sense to refactor the handler functions out and then on/off them specifically?

Reply:
Hey @kewisch, I've added namespaces to these events.